link/src/main/java/com/stripe/android/link/LinkPaymentLauncher.kt

```diff
@@ -239,6 +239,6 @@ class LinkPaymentLauncher @AssistedInject internal constructor(
     }
 
     companion object {
-        const val LINK_ENABLED = false
+        val LINK_ENABLED = BuildConfig.DEBUG
```

Comment:
Figured this would make development a little easier. Let me know if there are any concerns with this!